packages/in_app_purchase/lib/src/billing_client_wrappers/billing_client_wrapper.dart

```diff
@@ -195,6 +193,8 @@ class BillingClient {
       String? purchaseToken,
       ProrationMode? prorationMode}) async {
     assert(sku != null);
+    assert((oldSku == null) == (purchaseToken == null),
+        'oldSku and purchaseToken should have the same nullability');
```

Comment:
Nit: nullability is a property of a type, not a value; the two types do have the same nullability (both allow null). s/should have the same nullability/must both be set, or both be null/